script/Ciel/Day 13/13_01_01_CIEL13_1.txt

```diff
@@ -670,22 +721,23 @@ An impulse that drives me mad while keeping my will intact.
 [sha:e3d371906450c4846b7cebcbb3f6bbfa9e6061a3]{
 -- Page 35, Offset 33199.
 -- 人格や記憶は残したまま、目的―――欲求だけが少しずつ上書きされていく。
-While my personality and memories remain untouched, it's only my intentions―――my desires that are slowly being overwritten.
+It leaves my personality and memory untouched,
+ while slowly overwriting my intentions――my desires.
 }
 [sha:d61f4f4b63d43ebb0f1a013d987ab49f3524cad0]{
 -- Page 35, Offset 33200.
 -- 自分という意識を残したまま、違うものに変化していく恐怖。
-A fear of transforming into a different being while my own consciousness remains intact.
+I'm filled with the fear of remaining fully conscious as I transform into something else entirely.
 }
 [sha:16a0f1f2ce38cd27590d174e4e0b5bad6a549117]{
 -- Page 35, Offset 33201.
 -- これが。
-This is...
+This must be...
 }
 [sha:a4d0dfbd8844cdba86a68095bc89075acc20d8ab]{
 -- Page 35, Offset 33202.
 -- これがロアなのか。
-So this must be Roa.
+This must be Roa.
```

Comment:
These lines (and the one above which is unmarkable) is Shiki describing what Roa is, not what is happening to him.

>It is an urge that drives you mad while keeping your will intact.

>It leaves your personality and memory untouched, while slowly overwriting your purpose――your desires.

>It is the the horror of being transformed into something else while fully conscious of yourself.

>Is this it?

>Is this Roa?